docs/updating-components.md

```diff
@@ -0,0 +1,74 @@
+# How to update components?
+
+This document lists the steps to update components in Lokoctl.
+
+## Calico
+
+To update calico update the image tags in following files:
+
+- `assets/lokomotive-kubernetes/bootkube/resources/charts/calico/values.yaml`
+- `assets/lokomotive-kubernetes/bootkube/variables.tf`
+
+If there are changes necessary in the helm chart then make them in `assets/lokomotive-kubernetes/bootkube/resources/charts/calico`.
+
+Releases: https://github.com/projectcalico/calico/releases
+
+## metrics-server
+
+Run following commands from the root of this repository:
+
+```bash
+cd assets/components
+rm -rf metrics-server
+helm fetch --untar --untardir ./ stable/metrics-server
+```
+
+More information about the chart can be found in: https://github.com/helm/charts/tree/master/stable/metrics-server.
+Code repository: https://github.com/kubernetes-sigs/metrics-server
+
+## openebs
+
+Run following commands from the root of this repository:
+
+```bash
+cd assets/components
+rm -rf openebs
+helm fetch --untar --untardir ./ stable/openebs
+```
+
+Now create add a file in the chart:
```

Comment:
This isn't phrased correctly. I'm not sure what the intention here is.

Reply:
This was removed and the steps were conflated.